src/tests/core.py

```diff
@@ -320,15 +321,15 @@ def _decode_msg(data, pattern):
     def test_version(self):
         """check encoding/decoding of the version message"""
         # with single stream
-        msg = protocol.assembleVersionMessage('127.0.0.1', 8444, [1])
+        msg = protocol.assembleVersionMessage('127.0.0.1', 8444, [1], dandelion_ins.enabled)
```

Comment:
we don't really need an instance of dandelion here. We just need a variable and we pass the same variable to `assembleVersionMessage` and `assertEqual`

Reply:
Done